packages/apps/kadena-docs/src/components/Layout/components/Main/Main.tsx

```diff
@@ -35,7 +35,6 @@ export const Main: FC<IProps> = ({ children, frontmatter }) => {
       if (tree.length) {
         tree.map((item) => {
           // is the menu open?
```

Comment:
I'm aware this is not really part of the PR but, this function is relying on the side effect of mutating the item... It would be better to have the item returned in the map and used actually in a separate function. i.e.

```
tree.map(item => {
  return {
    ...item,
    isMenuOpen: pathname.startsWith(item.root),
    isActive: item.root === pathname,
 }
})// now use this tree to do something
```